.github/workflows/olm.yml

```diff
@@ -52,7 +52,7 @@ jobs:
 
     - name: Create and set up K8s Kind Cluster
       run: |
-        ./hack/kind-cluster-with-registry.sh
+        ./hack/kind-cluster.sh --kind-version v1.27.3 --cluster-config ./hack/kind-single-node-config.yaml --registry true
```

Comment:
I'd add a flag single node / multinode rather than exposing which config to load from outside.